request_a_govuk_domain/request/views.py

```diff
@@ -359,6 +361,64 @@ class MinisterUploadRemoveView(UploadRemoveView):
 class ConfirmView(TemplateView):
     template_name = "confirm.html"
 
+    def post(self, request):
+        """
+        Save the form and redirect to the success view.
+        :param request:
+        :return:
+        """
+        reference_ = request.POST["reference"]
+        try:
+            self.save_application_to_database_and_send_confirmation_email(
+                reference_, request
+            )
+        except IntegrityError as e:
+            logger.error(f"Exception while saving data. {type(e).__name__} - {str(e)}")
```

Comment:
This should be a warning saying trying to create application with existing reference?